prqlc/prqlc/Cargo.toml

```diff
@@ -8,6 +8,10 @@ repository.workspace = true
 rust-version = "1.70.0"
 version.workspace = true
 
+[[bin]]
+name = "prqlc"
+test = false
```

Comment:
Do we lose some tests here too? https://github.com/PRQL/prql/blob/8a230fc0f1d2a9c2f88682ab447d6ed28fecaea0/prqlc/prqlc/src/cli.rs#L505

Reply:
We do yes, similar to prqlc-parser